lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -52,6 +52,10 @@ function runA11yChecks() {
     },
     // @ts-ignore
   }).then(axeResult => {
+    // Scroll back to the top of the page so that element positions
+    // are relative to the top of the page
+    document.documentElement.scrollTop = 0;
```

Comment:
If I remember correctly axe is scrolling down the page during the run.

Reply:
ah if that's what it was I think we added the "scroll back to top" fix after this PR was created, so that might've been it

Reply:
`getBoundingClientRect` requires this, otherwise values are not relative to the top left of the document.

Reply:
@connorjclark I understand that we need to be scrolled to the top so the rect values are correct, I was trying to understand why we're not *already* at the top of the document because we thought we fixed the issue of axe scrolling. if it's broken, we should fix our fix

Reply:
We reset the scroll position in between each gatherer. For axe, we need the scroll reset to zero at the end of the gatherer. Our catch-all scroll reset is too late.

Reply:
Ah-ha gotcha! sorry! We need to scroll *in the middle of axe gatherer*. This point is after axe has already done its scrolling.

Reply:
looks like axe added an option to avoid scrolling:

https://github.com/ckundo/axe-core/commit/340705d211ce477ad5977616dad8aa4e4567e014
 
(it was only there for [some Firefox bug](https://github.com/dequelabs/axe-core/pull/148#issuecomment-164309022) anyway, it seems)

lets use `noScroll` instead of resetting?